_includes/navigation.html

```diff
@@ -53,7 +53,7 @@
             <a class="item" href="/articles/mobile/filtering.html">Customizing Content</a>
             <a class="item" href="/articles/mobile/authentication.html">Bridge Authentication</a>
             <a class="item" href="/articles/mobile/notifications.html">Notifications</a>
-            <a class="item" href="/articles/mobile/hosted_files.html">Hosted Files</a>
```

Comment:
If we're deleting this from the nav bar, are we also deleting it from our docs? If we're not deleting it from our docs, how are people going to access it?

Reply:
All the hosted file information should now be in this files page. I can delete the old page (I was just keeping it for reference).